docs/user-guide/patterns/rounding-envelopes.md

```diff
@@ -0,0 +1,102 @@
+# Rounding Envelopes
```

Comment:
again bullshit terminology

but this needs to be reviewed by someone working daily on FV of protocols